PyrusClient.cs

```diff
@@ -40,49 +52,56 @@ public class PyrusClient
 
 		static PyrusClient()
 		{
-			Settings = new Settings();
+			Settings = new Settings("https://api.pyrus.com/v4", "https://files.pyrus.com");
+		}
+
+		public PyrusClient(
```

Comment:
1. removed the parameterless constructor this will break backward compatibility
2. the Settings class has other parameters, let's do it like this

```c#
public PyrusClient() : this(Settings)
{
}

public PyrusClient(Settings settings)
{
	ClientSettings = settings ?? throw new ArgumentNullException(nameof(settings));;
}

Reply:
1. does it? optional parameters shouldn't affect backward compatibility
2. done

Reply:
this is a public class and someone could have used a parameterless constructor in their code